cartoframes/viz/legends/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""
+Legend helpers to generate legends faster.
+"""
+
+from __future__ import absolute_import
```

Comment:
This is no longer required (py27 :fire:)

Reply:
I was going to delete them all, but I think it'd be better to do this in a separated task. I'll open a PR later 👍

Reply:
Those are already removed in https://github.com/CartoDB/cartoframes/pull/1383

Reply:
So I wouldn't add more to be removed in other PR.

Reply:
Ok! Didn't remember they were removed 👍fixed :)